oe_authentication.install

```diff
@@ -32,7 +32,7 @@ function oe_authentication_install(): void {
       // We need to add a default email domain so that we can trigger
       // the appropriate event to handle emails.
       ->set('user_accounts.email_assignment_strategy', CasUserManager::EMAIL_ASSIGNMENT_STANDARD)
-      ->set('user_accounts.email_hostname', 'localhost')
+      ->set('user_accounts.email_hostname', 'example.com')
```

Comment:
How come the post update is not here?

Reply:
Post update implemented in EWCMS.

Reply:
@sergepavle Yes, I know! Why is it there? Why not in the component where we are changing it on install?